lib/phlex/html.rb

```diff
@@ -187,7 +187,8 @@ def text(content)
 				case content
 					when String then content
 					when Symbol then content.name
-					else content.to_s
+					when Integer, Float then content.to_s
+					else format_object(content) || content.to_s
```

Comment:
We fall back to `content.to_s` here because this object has been intentionally passed to `text` meaning we know the developer wants to output something. We don’t do this in `yield_content` becuase that would result in potentially calling `to_s` on an empty Enumeratorand outputting an Enumerator object description.